canonical/lxd-ui
# Ensure multiselect snapshots behaves the same as instances WD-6704

## Done

- Ensure multiselect snapshots behaves the same as instances

Fixes WD-6704

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - open an instance detail page > snapshots
    - create some snapshots
    - ensure multiselect works as expected
    - multi delete some snapshots
